modules/imaging_browser/help/imaging_browser.md

```diff
@@ -0,0 +1,36 @@
+# Imaging Browser
+
+The Imaging Browser allows users to browse and view candidate scan
+sessions collected at time-points (visits) across the study.
+
+## Searching for a candidate's scans
+
+Use the Selection Filter panel to specify search criteria such as
+candidate PSCID, DCCID, Visit Label, Site, Quality Control (QC)
+status of the scanning session, and any sequence type (T1, T2, DTI)
+in the scan session.  As well, users may filter for scans that are
+New (have not had any QC done), Pending (QC is not finalized) or both.
+
+After specifying search criteria in the selection filters, click
+"Show Data" to display results in the data table below, with one
+row per relevant scanning session. Click "Clear Form" to reset all
+selection filters.
+
+Sort the data table results by clicking on any column header.  The
+date of first acquisition of all scans for the timepoint is indicated
+under the "First Acq Date" column.  The "Last QC" column shows the
```

Comment:
The column is now called 'First Acquisition"

Reply:
done